src/hooks/useHandleTransfer.tsx

```diff
@@ -283,58 +308,250 @@ async function evm(
+
+        try {
+          const L2WormholeGateway = new Contract(
+            sourceAddress,
+            ThresholdL2WormholeGateway,
+            signer
+          );
+
+          console.log("SEND TBTC?!?");
+
+          const amountNormalizeAmount = deNormalizeAmount(
+            normalizeAmount(transferAmountParsed, decimals),
+            decimals
+          );
+
+          console.log({
+            signer,
+            transferAmountParsed,
+            amountNormalizeAmount,
+            recipientChain,
+            readableTargetAddress,
+          });
+
+          console.log(
+            "zeroPad(arrayify(targetAddress), 32)"
+            // zeroPad(arrayify(targetAddress), 32)
+          );
+
+          const estimateGas = await L2WormholeGateway.estimateGas.sendTbtc(
+            amountNormalizeAmount,
+            recipientChain,
+            zeroPad(arrayify(readableTargetAddress!), 32),
+            THRESHOLD_ARBITER_FEE,
+            THRESHOLD_NONCE
+          );
+
+          // We increase the gas limit estimation here by a factor of 10% to account for
+          // some faulty public JSON-RPC endpoints.
+          const gasLimit = estimateGas.mul(1100).div(1000);
+          const overrides = {
+            gasLimit,
+            // We use the legacy tx envelope here to avoid triggering gas price autodetection using EIP1559 for polygon.
+            // EIP1559 is not actually implemented in polygon. The node is only API compatible but this breaks some clients
+            // like ethers when choosing fees automatically.
+            ...(chainId === CHAIN_ID_POLYGON && { type: 0 }),
+          };
+
+          console.log("sendTbtc overrides", { overrides });
+
+          const tx = await L2WormholeGateway.sendTbtc(
+            amountNormalizeAmount,
+            recipientChain,
+            zeroPad(arrayify(readableTargetAddress!), 32),
+            THRESHOLD_ARBITER_FEE,
+            THRESHOLD_NONCE,
+            overrides
+          );
+
+          const receipt = await tx.wait();
+          console.log({ receipt });
+
+          dispatch(
+            setTransferTx({
+              id: receipt.transactionHash,
+              block: receipt.blockNumber,
+            })
+          );
+
+          enqueueSnackbar(null, {
+            content: <Alert severity="success">Transaction confirmed</Alert>,
+          });
+
+          const sequence = parseSequenceFromLogEth(
+            receipt,
+            getBridgeAddressForChain(chainId)
+          );
+          const emitterAddress = getEmitterAddressEth(
+            getTokenBridgeAddressForChain(chainId)
+          );
+
+          console.log({ sequence, emitterAddress });
+
+          await fetchSignedVAA(
+            chainId,
+            emitterAddress,
+            sequence,
+            enqueueSnackbar,
+            dispatch
+          );
+
+          // const { vaaBytes, isPending } = await getSignedVAAWithRetry(
```

Comment:
does it need to be commented on? A comment explaining why this comment will be use would be good to have

Reply:
I forgot to delete it! done ✅